task_DEV_2/task_DEV_2/Program.cs

```diff
@@ -6,48 +6,65 @@ class Program
+            Console.WriteLine("Enter the length of the array :");
             int arrayLength = Convert.ToInt32(Console.ReadLine());
 
             if (arrayLength <= 20)
             {
-                int[] inputNumbers = new int[arrayLength];
+                int[] inputNumbers = CreateNumbersArray(arrayLength);
+                PrintNumberArraryToConsole(inputNumbers, arrayLength);
+                SwapMinMaxElements(inputNumbers, arrayLength);
+                PrintNumberArraryToConsole(inputNumbers, arrayLength);
+            }
+            else
+            {
+                Console.WriteLine("array cannot be > 20");
+            }
+        }
 
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                int max = inputNumbers[0];
-                int min = inputNumbers[0];
-                int minIndex = 0;
-                int maxIndex = 0;
+        static int CreateNumbersArray(int arrayLength)
+        {
+            int[] inputNumbers = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            return inputNumbers;
+        }
+
+        static int[] SwapMinMaxElements(int[] inputNumbers, int arrayLength)
+        {
+            int max = inputNumbers[0];
+            int min = inputNumbers[0];
+            int minIndex = 0;
+            int maxIndex = 0;
 
-                for (int i = 0; i < arrayLength; i++)
+            for (int i = 0; i < arrayLength; i++)
+            {
+                if (inputNumbers[i] > max)
                 {
-                    if (inputNumbers[i] > max)
-                    {
-                        max = inputNumbers[i];
-                        maxIndex = i;
-                    }
-                    if (inputNumbers[i] < min)
-                    {
-                        min = inputNumbers[i];
-                        minIndex = i;
-                    }
-                    Console.WriteLine(inputNumbers[i]);
+                    max = inputNumbers[i];
+                    maxIndex = i;
                 }
-                inputNumbers[maxIndex] = min;
-                inputNumbers[minIndex] = max;
-
-                for (int i = 0; i < arrayLength; i++)
+                if (inputNumbers[i] < min)
                 {
-                    Console.WriteLine(inputNumbers[i]);
+                    min = inputNumbers[i];
+                    minIndex = i;
                 }
+                Console.WriteLine(inputNumbers[i]);
             }
-            else
+            inputNumbers[maxIndex] = min;
+            inputNumbers[minIndex] = max;
+
+            return inputNumbers[minIndex];
```

Comment:
Why does this method return min element? This method has to return inputNumbers.

Reply:
done